pkg/echo/README.md

```diff
@@ -21,6 +21,26 @@ func main() {
 }
 ```
 
+### Per-request logger
+
+It is possible to set individual echo proxy for each request which performs context passing into every single logging record automatically. Just make sure to put middleware that sets context fields before echo proxy middleware:
```

Comment:
Let's write down here that this logs the entire context; including anything that might be secret and that sensitive values should not be added to the default context?